src/main/scala/devices/debug/Debug.scala

```diff
@@ -1424,10 +1426,13 @@ class TLDebugModuleInnerAsync(device: Device, getNComponents: () => Int, beatByt
     withClock (gated_clock) {
       dmInner.module.clock := gated_clock
       dmInner.module.io.dmactive := dmactive_synced
-      dmInner.module.io.innerCtrl := FromAsyncBundle(io.innerCtrl)
+      withReset (~dmactive_synced) {
+        dmInner.module.io.innerCtrl := FromAsyncBundle(io.innerCtrl)
+      }
       dmInner.module.io.debugUnavail := io.debugUnavail
       io.hgDebugInt := dmInner.module.io.hgDebugInt
       io.extTrigger.foreach { x => dmInner.module.io.extTrigger.foreach {y => x <> y}}
+      dmiXing.module.reset := false.B  // Safe AsyncQueue is reset from DMI side only
```

Comment:
Why is this not dmactive_synced as well?

Reply:
The reason is that this would prevent access to dmstatus when dmactive=0.  It's useful to be able to read the version before setting dmactive to 1.  My understanding of the Safe Async Queue is that only one side needs a reset because there is logic to reset both sides when either side's reset is asserted.

Reply:
I see. The initial design was intended to prevent traffic to go across the crossing when periphery was in reset, but this is a conscious decision to change that design, so this makes sense. We just need to ensure that the regmapper does not need running clock or to be out of reset in order to respond properly.